pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
On a related note, I don't think we do any caching of courses (see, e.g., [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script.rb#L194-L216)) (CC @breville). Is this something we should be doing?